src/engine/sidechain/shoutconnection.cpp

```diff
@@ -13,7 +13,7 @@
 #ifdef WIN64
 #define WIN32
 #endif
-#include <shout/shout.h>
+#include "../lib/libshout/include/shout/shout.h"
```

Comment:
The include directives must not change for bundled libraries to allow replacing a bundled library with a (newer) system library!

Reply:
This is just a quick hack to be sure the expected header is used. I think we only need to actually merge this if we discover a libshout bug.
For now I think we can use libshour_idjc to enable aac.